src/components/Dropdown/index.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import InputLabel from '@material-ui/core/InputLabel';
+import MenuItem from '@material-ui/core/MenuItem';
+import FormControl from '@material-ui/core/FormControl';
+import Select from '@material-ui/core/Select';
+import classNames from 'classnames';
+import dropdownStyles from './styles';
+
+interface OptionItem {
+  id: number,
+  value: string
+}
+
+interface OptionsListProps {
+  options: OptionItem[],
+}
+
+interface DropdownProps {
+  name: string,
+  options: OptionItem[],
+  title: string,
+  className?: string,
+}
+
+const OptionsList: React.FC<OptionsListProps> = (props) => {
+  
+  const { options } = props;
+
+  if (options && options.length) {
+    return (
+      <div>
+        {options.map((opt) => (
+          <MenuItem key={opt.id} value={opt.id}>{opt.value}</MenuItem>
+        ))}
+      </div>
+    );
+  }
+
+  return null;
```

Comment:
FYI> Not a big deal but you can do something like 
```
 if (!(options && options.length)) return null
```

in the begining of this function, so the js enginge avoids the computing of extra lines in this case. \